Demo/src/main/java/com/braintreepayments/demo/VenmoFragment.java

```diff
@@ -86,12 +91,12 @@ public void launchVenmo(View v) {
                 lineItems.add(new VenmoLineItem(VenmoLineItem.KIND_DEBIT, "Two Items", 2, "10"));
                 venmoRequest.setLineItems(lineItems);
 
-                venmoClient.createPaymentAuthRequest(requireActivity(), venmoRequest, (venmoAuthChallenge, authError) -> {
-                    if (authError != null) {
-                        handleError(authError);
-                        return;
+                venmoClient.createPaymentAuthRequest(requireActivity(), venmoRequest, (paymentAuthRequest) -> {
+                    if (paymentAuthRequest instanceof VenmoPaymentAuthRequest.Failure) {
```

Comment:
Do we still think `VenmoPaymentAuthRequest` is the best naming or is it too wordy for the Java integration?

Reply:
I think it's fine unless you had ideas on another name, though we probably would want to update everything not just Venmo if so.